From 98c817a7ef9afc467541a459bc875c24003da5e5 Mon Sep 17 00:00:00 2001
From: Olivier Masse <olivier.masse@nxp.com>
Date: Mon, 18 Nov 2019 17:05:17 +0100
Subject: [PATCH 48/52] MMIOT-192: Remove CONFIG_MXC_HANTRO_SECURE by default

CONFIG_MXC_HANTRO_SECURE is optionally added in an additionnal defconfig file

Signed-off-by: Olivier Masse <olivier.masse@nxp.com>
---
 arch/arm64/configs/defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 25d33e8e4bab..10008cdcac47 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -639,7 +639,6 @@ CONFIG_MMC_SDHCI_XENON=y
 CONFIG_MXC_MLB150=y
 CONFIG_MXC_SIM=y
 CONFIG_MXC_EMVSIM=y
-CONFIG_MXC_HANTRO_SECURE=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
-- 
2.17.1

